setup.cfg

```diff
@@ -26,7 +26,8 @@ install_requires =
   thefuzz
   matplotlib>=3.3.3
   timple>=0.1.2
-  signalr-client-aio
+  requests>=2.28.0
```

Comment:
One minor thing here. `requests` is already defined above without a version constraint. We should define it twice. After that, this looks good to be merged

Reply:
@theOehrly No problem. Fixed.